### DISCUSSION TITLE
crowdsec

### DISCUSSION THREAD

**Discussion by captainabloc** [ZoeyVid/NPMplus#660] [Q&A] [answered]
Hello,
thanks for this NPM fork

regarding crowdsec integration, would someone have more explanation how to install and then add plugins (where to find them and modifications to do on conf file).

I installed crowdsec using the docker compose file given, are we supposed to see anything on local_IP: 8080?

Also, can it be linked to crowdsec dashboard?

thanks

**Comment by Zoey2936** [comment, marked as answer]
ip:8080 is just an API, for anything else please read the crowdsec docs
